doc/chapters/api/index.rst

```diff
@@ -24,88 +13,276 @@ Methods
 
 Ping
 ----
-Ping()
+Controls availability of com.pelagicore.SoftwareContainerAgent interface
+
+:Parameters:
+        - **None**
+
+:Return Value:
+        - **None** 
+
+:Example Usage:
+
+::
+
+        dbus-send --system --print-reply \
+        --dest=com.pelagicore.SoftwareContainerAgent \
+        /com/pelagicore/SoftwareContainerAgent \
+        com.pelagicore.SoftwareContainerAgent.Ping
```

Comment:
I think it would be enough to document the API and have the usage somewhere else. Perhaps add missing parts to the Getting started chapter, or just reference it from this chapter? I'm mainly concerned about maintenance.

I don't think we need to explain dbus-send much, just point out what parts of the command is the object path etc., that should be enough for the user to figure out any details about dbus-send.